internal/testplanet/planet.go

```diff
@@ -148,7 +148,7 @@ func New(t zaptest.TestingT, satelliteCount, storageNodeCount, uplinkCount int)
 			Path:               storageDir,
 			AllocatedDiskSpace: memory.GB.Int64(),
 			AllocatedBandwidth: 100 * memory.GB.Int64(),
-		}, node.Identity.Key)
+		}, node.Identity.Key, zap.S())
```

Comment:
Change to a named logger
`zap.L().Named(fmt.Sprintf("storage node (%s)", node.Identity().ID))`

Reply:
the testplanet node has a named logger. I switched to that instead